Stripe/STPPickerTableViewCell.h

```diff
@@ -0,0 +1,30 @@
+//
+//  STPPickerTableViewCell.h
+//  Stripe
+//
+//  Created by Ben Guo on 2/9/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "STPTextFieldTableViewCell.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol STPPickerDataSource;
+
+@interface STPPickerTableViewCell : STPTextFieldTableViewCell
+
+@property(nonatomic) id<STPPickerDataSource> pickerDataSource;
+
+@end
+
+@protocol STPPickerDataSource <NSObject>
+
+- (NSInteger)numberOfRows;
```

Comment:
Do we want to name these something more specific to picker data source to avoid potential future method name collisions?